### PR TITLE
Added the Test cases as per the SonarQube report.

### PR DESCRIPTION
**Pull request by AVandana-OSI**
Test Case for AssignmentServiceTest, ClientAddressTest and Deleted the postedTickle and some cleanup.

### CODE REVIEW

File: src/main/java/gov/ca/cwds/rest/services/cms/ClientAddressService.java [hunk 1]
```diff
@@ -77,12 +77,17 @@ public Response find(Serializable primaryKey) {
     return null;
   }
 
+  /**
+   * @param address - aadress
```
Comment (tparkerosi) [inline src/main/java/gov/ca/cwds/rest/services/cms/ClientAddressService.java]: spelling error

File: src/test/java/gov/ca/cwds/fixture/ClientAddressResourceBuilder.java [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+  String effEndDt = "2016-01-05";
+  String effStartDt = "2017-01-05";
+  String fkAddress = "FKADDRESSX";
+  String fkClient = "FKCLIENTXX";
+  String homelessInd = "";
+  String fkReferral = "FKREFERRAL";
+
+  /**
+   * @return the ClientAddress
+   */
```
Comment (tparkerosi) [inline src/test/java/gov/ca/cwds/fixture/ClientAddressResourceBuilder.java]: Typically these methods are called build instead of createXxxxx

Comment (AVandana-OSI) [inline src/test/java/gov/ca/cwds/fixture/ClientAddressResourceBuilder.java]: So need to change the create*** to build***

File: src/test/java/gov/ca/cwds/rest/services/cms/AssignmentServiceTest.java [hunk 3]
```diff
@@ -0,0 +1,316 @@
...
+    staffPersonIdRetriever = mock(StaffPersonIdRetriever.class);
+    assignmentService = new AssignmentService(assignmentDao, staffPersonIdRetriever);
+  }
+
+  // find test
+  @Override
+  @Test(expected = AssertionError.class)
+  public void testFindThrowsAssertionError() {
+    // thrown.expect(AssertionError.class);
+    try {
```
Comment (tparkerosi) [inline src/test/java/gov/ca/cwds/rest/services/cms/AssignmentServiceTest.java]: We should probably update the test name to include that the thrown error has a message. For example assignmentServiceFindMethodThrowsAssertionErrorWithMessage, testAssertionErrorContainsMessage, or something similar. The test name should describe what is being tested.

File: src/test/java/gov/ca/cwds/rest/services/cms/AssignmentServiceTest.java [hunk 4]
```diff
@@ -0,0 +1,316 @@
...
+  @Override
+  @Test
+  public void testDeleteReturnsNullWhenNotFound() throws Exception {
+    Response found = assignmentService.delete("ABC1234567");
+    assertThat(found, is(nullValue()));
+  }
+
+  @Override
+  public void testDeleteThrowsNotImplementedException() throws Exception {
+
```
Comment (tparkerosi) [inline src/test/java/gov/ca/cwds/rest/services/cms/AssignmentServiceTest.java]: remove empty tests

File: src/test/java/gov/ca/cwds/rest/services/cms/ClientAddressServiceTest.java [hunk 5]
```diff
@@ -27,21 +47,34 @@
...
+  /**
+   * 
+   */
   @Test
-  public void shouldReturnNullWhenNotClientAddressesAreFound(){
-    when(clientAddressDao.findByAddressAndClient(any(), any())).thenReturn(new ArrayList());
-//    clientAddressService = new ClientAddressService(clientAddressDao, staffpersonDao, triggerTablesDao,laCountyTrigger, staffPersonIdRetriever);
+  public void shouldReturnNullWhenNotClientAddressesAreFound() {
+    when(clientAddressDao.findByAddressAndClient(any(), any())).thenReturn(new ArrayList<>());
+    // clientAddressService = new ClientAddressService(clientAddressDao, staffpersonDao,
```
Comment (tparkerosi) [inline src/test/java/gov/ca/cwds/rest/services/cms/ClientAddressServiceTest.java]: remove comments

**Pull request merged**